src/main/port.ts

```diff
@@ -16,5 +17,6 @@ export async function getFreePort(): Promise<number> {
       logger.error(`Can't resolve new port: "${error}"`);
       reject(error);
     });
+    server.listen({ host: "127.0.0.1", port: 0 })
```

Comment:
It seems that we need to register callbacks before starting listening to port. Also brought back `host` option.